AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -1152,6 +1123,173 @@ msgstr "Green Level"
 msgid "Casos Notificados de %(disease_label)s"
 msgstr "Reported cases of %(disease_label)s"
 
+#: dados/templates/services.html:19
+msgid "Serviços de Dados Públicos"
+msgstr "Public Data Service"
+
+#: dados/templates/services_api.html:69
+#: dados/templates/services_api_doc.html:46
+#: dados/templates/services_maps.html:69
+#: dados/templates/services_maps_doc.html:47 templates/base.html:74
+msgid "Dados"
+msgstr "Data"
+
+#: dados/templates/services_api.html:69
+msgid "API"
+msgstr ""
+
+#: dados/templates/services_api.html:79
+msgid "Para mais informações, acesse a documentação do serviço da API."
+msgstr "For more information, access the map service documentation"
+
+#: dados/templates/services_api.html:84
+msgid "Filtro Município (código IBGE, nome do município ou U.F.)"
+msgstr ""
+
+#: dados/templates/services_api.html:88
+msgid "Digite aqui o filtro desejado para a consulta de municípios"
+msgstr ""
+
+#: dados/templates/services_api.html:93
+msgid ""
+"Digite o código IBGE, nome do município ou U.F. para buscar o município "
+"desejado."
+msgstr "Type the IBGE code or fragment to search available IBGE code."
+
+#: dados/templates/services_api.html:103 dados/templates/services_maps.html:103
+#: dados/templates/state_cities.html:198
+msgid "Selecione municipio(s)"
+msgstr "Select one or more cities"
+
+#: dados/templates/services_api.html:109
+msgid "Selecione o município desejado"
+msgstr "Select the desired city to search"
+
+#: dados/templates/services_api.html:119
+msgid "Data da Semana Epidemiológica de início da consulta"
+msgstr "Date of the epidemiological week"
+
+#: dados/templates/services_api.html:126
+msgid "Selecione um dia da Semana Epidemiológica de início da consulta"
+msgstr "Select a day from the desired epidemiological week "
+
+#: dados/templates/services_api.html:131 dados/templates/services_api.html:159
+msgid "Data da Semana Epidemiológica de término da consulta"
+msgstr "Date of the epidemiological week"
```

Comment:
the translation should explain that the date is related to the last week of the search.

Reply:
ok